src/routes/mod.rs

```diff
@@ -114,3 +128,8 @@ fn no_auth(_req: &Request) -> Authenticate {
 fn login(auth_user: ValidBasicToken, tc: State<TrowConfig>) -> Result<TrowToken, Error> {
     trow_token::new(auth_user, tc).map_err(|_| Error::InternalError)
 }
+
+#[options("/login")]
```

Comment:
Do we really need to add options everywhere? Can't we do it in the fairing? I saw https://github.com/SergioBenitez/Rocket/issues/25#issuecomment-313895086